src/main/java/com/mercuryCyclists/Sales/controller/InStoreSaleController.java

```diff
@@ -0,0 +1,71 @@
+package com.mercuryCyclists.Sales.controller;
+
+import com.mercuryCyclists.Sales.entity.InStoreSale;
+import com.mercuryCyclists.Sales.service.InStoreSaleService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@CrossOrigin
+@RestController
+@RequestMapping(path = "api/v1/in-store-sale")
+public class InStoreSaleController {
+
+    private final InStoreSaleService inStoreSaleService;
+
+    @Autowired
+    public InStoreSaleController(InStoreSaleService inStoreSaleService) {
+        this.inStoreSaleService = inStoreSaleService;
+    }
+
+    /**
+     * Gets all In Store Sales
+     */
+    @GetMapping
+    public List<InStoreSale> getInStoreSales() {
+        return inStoreSaleService.getAllInStoreSales();
+    }
+
+    /**
+     * Gets In Store Sales by Store ID
+     */
+    @GetMapping(path = "/store/{storeId}/sales")
+    public List<InStoreSale> getStoreSales(@PathVariable("storeId") Long storeId) {
+        return inStoreSaleService.getStoreSales(storeId);
+    }
```

Comment:
Thinking more, mabye we should put this in the store controller but direct it to the inStoreSaleService?

Reply:
Seems a bit chank calling a diff service from a controller (store controller calling in store sale service)